content/press/36-civil-rights-groups-demand-end-to-amazons-partnerships-with-police/contents.lr

```diff
@@ -0,0 +1,19 @@
+_model: post
+---
+_hidden: yes
+---
+active: True
+---
+type: snippet
+---
+publisher:  VICE
+---
+title:  36 Civil Rights Groups Demand End to Amazon’s Partnerships with Police
+---
+link:  https://www.vice.com/en_us/article/ne8q3w/36-civil-rights-groups-demand-end-to-amazons-partnerships-with-police?utm_source=mbtwitter
```

Comment:
i think we could take the twitter suffix away, '?utm_source=mbtwitter'

Reply:
Thanks :), all done @emmapeel2